cmake/Util.cmake

```diff
@@ -30,6 +30,7 @@ function(install_headers src dst)
                           FILES_MATCHING PATTERN "*.h"
                                          PATTERN "*.hpp"
                                          PATTERN "*.hh"
+                                         PATTERN "3rdparty*" EXCLUDE
```

Comment:
Not sure how/if it was possible to include e.g., installed runtime headers in user code, and nothing seems to have changed here with this commit. Will we eventually need code here to install 3rdparty deps, or should we instead never use 3rdparty headers in public headers?